core/src/main/resources/jenkins/install/SetupWizard/setupWizardFirstUser.jelly

```diff
@@ -44,6 +51,29 @@
           <script>
             $('username').focus();
           </script>
+       </form>
+       <form action="${rootURL}/setupWizard/configureRootUrl" class="root-url no-json" method="post">
```

Comment:
`saveRootUrl` specifies the URL to post to, so I suspect `action` and `method` could just be deleted.

Reply:
removed